theSingularity.js

```diff
@@ -0,0 +1,389 @@
+  choices = readlineSync.keyInSelect(array, "HOW WOULD YOU ESCAPE?");
+  playerChoice = array[choices];
+  if (playerChoice === "KEYS") {
+    console.log(playerChoice);
+    console.log(`GREAT! YOU CAN FREE YOUR HANDS AND FEET NOW! RUN!`);
+    goSomewhere();
+  } else if (playerChoice === "GUN") {
+    console.log(playerChoice);
+    console.log(`YOU REACHED FOR THE UNCONCIOUS GUARD'S GUN BUT THE OTHER
+GUARD HAS SPOTTED YOU!
+YOU WERE CAPTURED !
+      **************
+      **GAME OVER!**
+      **************`);
+    if (readlineSync.keyInYN("Play Again?")) {
+      play();
+    } else console.log("This place is not for everyone!");
+    quitGame();
+  } else if (playerChoice === "RUN") {
+    console.log(playerChoice);
+    console.log(
+      `YOU ARE STILL IN CHAINS AND CAN'T RUN FAST ENOUGH. THE GIARD CAUGHT UP TO YOU
+      **************
+      **GAME OVER!**
+      **************`
+    );
+
+    if (readlineSync.keyInYN("Play Again?")) {
+      play();
+    } else console.log("THIS PLACE IS NOT FOR EVERYONE");
+    quitGame();
+  } else if (playerChoice === undefined) {
+    quitGame();
+  }
+};
+
+const goSomewhere = () => {
+  let directionInput = readlineSync.question(
+    "What direction would you like to take?\n "
+  );
+  if (directionInput === "north") {
+    console.log(`
+ WELCOME TO THE CITY OF NOWHERE!
+ YOU ARE NOT SUPPOSE TO BE HERE
+ THERE IS A SHELTER 40 FT AWAY
+ SCAVNAGERS MIGHT LIVE THERE, BECAREFUL!
+ YOU CAN ALWAYS GO BACK!
+     `);
+    directionNorth();
+  } else if (directionInput === "south") {
+    console.log(`
+  LOOKS LIKE YOU REACHED THE VAST DESERT.
+  NO SIGNS OF LIFE HERE AND A SAND STORM IS FORMING.
+  THERE IS A CAVE 500 FT FURTHER SOUTH. IT MIGHT BE THE
+  PEREFECT HIDEOUT FOR A REPTILIAN & YOU ARE UNARMED.
+  50 FT TO YOUR LEFT THERE IS AN ABANDONED CABIN. IT WAS
+  BUILT BY HUMANS. MIGHT FIND WATER. 
+  `);
+    directionSouth();
+  } else if (directionInput === "east") {
+    console.log(`
+    WELCOME TO THE OUTSIDE OF SUN CITY!
+    THE ONLY CITY WERE FULLY ORGANIC BEINGS RESIDE
+    IT IS A BEAUTIFUL PLACE!
+    BUT IS A FORTRESS. HOW ARE YOU GOING TO GET IN?`);
+    directionEast();
+  } else if (directionInput === "west") {
+    console.log("West then!");
+    console.log("WELCOME TO THE WILD WILD WEST!");
+    console.log("THE CITY OF REJECTS, OUTDATED A.I AND REPTILIANS");
+    console.log("YOU'll COME ACROSS SOME WAYS TO EARN RESOURCES");
+    console.log("BUT FIRST, YOU HAVE TO PROVE YOUR SELF");
+    console.log("MR T-REX, A REPTILIAN IS CHALLENGING YOU TO A FIGHT ");
+    console.log("HE REMAINS THE CHAMPION AND GUARDIAN OF SCRAP CITY!");
+    console.log(
+      "IF YOU WIN, YOU GET ACCESS TO THE CITY. IF YOU LOOSE YOU DON'T ENTER THE CITY"
+    );
+    directionWest();
+  } else console.log("NOT A VALID DIRECTION");
+  goSomewhere();
+};
+
+const directionNorth = () => {
+  let whereInNorth = readlineSync.question(
+    "Where would you like to go now?\n "
+  );
```

Comment:
suggestion would be to put the options for the user. I got stuck here and had to look at the code. 
You should also take the users input and manipulate it to make sure it matches like:
whereInNorth = whereInNorth.toLowerCase().trim() 
This ensures that I can write SHelter in any way and still have it work.